pom.client.xml

```diff
@@ -402,6 +404,10 @@
                 <title>Azure Storage - Blobs</title>
                 <packages>com.azure.storage.blob*</packages>
               </group>
+              <group>
+                <title>Azure Storage - Blobs Cryptography</title>
```

Comment:
I'd personally put a hyphen after 'Blobs' (and same with the Queues line below).

Reply:
Done